### PR TITLE
DHFPROD-3023: Test button checks for errors in UI mapping

### PR DESCRIPTION
**Pull request by xnikhil08**
Contains changes for the UI portion of story DHFPROD-3023

### CODE REVIEW

File: web/src/main/ui/app/components/flows-new/edit-flow/mapping/mapping.component.ts [hunk 1]
```diff
@@ -77,6 +78,48 @@ export class MappingComponent implements OnInit {
...
+            "sourcedFrom": "Shrivastava",
+            "errorMessage": "Invalid XPath expression: lastname"
+          },
+          "gender": {
+            "sourcedFrom": "Male"
+          }
+        }
+      }
+    }
+  };
```
Comment (srinathgit) [inline web/src/main/ui/app/components/flows-new/edit-flow/mapping/mapping.component.ts]: Mapping validation works for nested entities as well . The code was merged to develop. This hardcoded response  can be removed and you can check if it works for nested entities .

Comment (xnikhil08) [reply web/src/main/ui/app/components/flows-new/edit-flow/mapping/mapping.component.ts]: @srinathgit - Please review now. That code is removed.

**Comment by wooldridge** [comment]
There will most likely be tweaks required to mapping tests, but let's merge this so we can get what's current in mapping into develop.

**Comment by ryanjdew** [comment]
Looks like there is an error in the UI build.

`ERROR in src/main/ui/app/components/flows-new/edit-flow/mapping/mapping.component.ts.MappingComponent.html(15,7): : Property 'mapValidationResult' does not exist on type 'MappingComponent'.`

**Comment by wooldridge** [comment]
@xnikhil08 can you fix the build issue and push to this branch?

**Comment by xnikhil08** [comment]
> @xnikhil08 can you fix the build issue and push to this branch?

Fixed it.

**Pull request merged**